### PR TITLE
#159361806 easily onboard customers

### PR DESCRIPTION
**Pull request by phemonick**
#### What does this PR do?
easily onboard customers to the app
#### Description of Task to be completed?
User should see a modal when visiting the site for the first time.
 Modal should show a link to the FAQ page.
 A well documented and detailed FAQ page should be available to help customers onboard easily.
#### How should this be manually tested?
npm test
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#35 
#### Screenshots or gifs (if appropriate):
screen shot 2018-08-24 at 12 16 13 pm

#### Questions:

### CODE REVIEW

File: imports/plugins/custom/faq/client/components/faq.js [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+            <dt>
+              <button data-toggle="collapse" aria-expanded="false"
+                data-target="#demo2"
+              >How do i Ship my order?</button>
+            </dt>
+            <dd id="demo2" className="collapse">
+              <p>Enter your shipping address or select a saved address <br/>
+                  Choose the shipping speed for your items based on the
+                   arrival dates shown. If no options are available, it's
```
Comment (Stylll) [inline imports/plugins/custom/faq/client/components/faq.js]: You should try escaping the single quote used here.

File: imports/plugins/custom/faq/client/components/faq.js [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+              <button data-toggle="collapse" aria-expanded="false"
+                data-target="#demo3"
+              >Special offers?</button>
+            </dt>
+            <dd id="demo3" className="collapse">
+              <p>It is a long established fact that a reader will be distracted
+                 by the readable content of a page when looking at its layout.
+                  The point of using Lorem Ipsum is that it has a more-or-less
+                   normal distribution of letters, as opposed to using
+                    'Content here, content here
```
Comment (Stylll) [inline imports/plugins/custom/faq/client/components/faq.js]: Why is there a "Content here, content here" text here ?

File: imports/plugins/custom/faq/client/components/faq.js [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+            <dt>
+              <button data-toggle="collapse" aria-expanded="false"
+                data-target="#demo10"
+              >Payment on delivery?</button>
+            </dt>
+            <dd id="demo10" aria-expanded="false" className="collapse">
+              <p>Just place your order online, select "Pay On Delivery" as your
+                 payment option, and pay for your order when its brought to you.
+                  If you don't need to pay with cash, but just prefer to pay
```
Comment (Stylll) [inline imports/plugins/custom/faq/client/components/faq.js]: You should try escaping the single quote used here.

File: imports/plugins/custom/faq/client/components/faq.js [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+            <dt>
+              <button data-toggle="collapse" aria-expanded="false"
+                data-target="#demo2"
+              >How do i Ship my order?</button>
+            </dt>
+            <dd id="demo2" className="collapse">
+              <p>Enter your shipping address or select a saved address <br/>
+                  Choose the shipping speed for your items based on the
+                   arrival dates shown. If no options are available, it's
+                   because your item has only one shipping option
```
Comment (ozimos) [inline imports/plugins/custom/faq/client/components/faq.js]: You need to escape the apostrophe in your html. Use the &#39; code for apostrophe. Also edit the content to be more relevant

File: imports/plugins/custom/faq/client/stylesheets/index.less [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+.faq-content {
+  width: 100%;
+
+  h2 {
+    font-size: 32px;
+    font-style: normal;
+    font-weight: bold;
+    text-transform: uppercase;
+    line-height: 1.5;
+    color: #000000;
```
Comment (Stylll) [inline imports/plugins/custom/faq/client/stylesheets/index.less:17]: I believe hex codes should be stored in variables and referenced using their variable names. It's not really important but you should try storing the hex codes in variables and reference them with their names if time permits.

**Review by Stylll — COMMENTED** [review]
Great job!. Please check out my comments and make amendments where necessary.

**Review by Stylll — APPROVED** [review]
Good Job.

**Pull request merged**